cypress/integration/e2e/theme-switcher/index.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('Theme switcher', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('check the console theme radio', () => {
+    cy.get('input[name="sitetheme"]')
+      .check('console')
+      .should('be.checked');
+  });
+
+  it('check the default theme radio', () => {
+    cy.get('input[name="sitetheme"]')
+      .check('default')
+      .should('be.checked');
```

Comment:
I don't necessarily see how these cam both be checked

Reply:
They're two separate tests. First one loads the site, and basically checks to see if the console theme is checked. The second one checks the default theme and verifies its checked.

Reply:
oh I see now